tarpc/src/server/in_flight_requests.rs

```diff
@@ -11,29 +11,40 @@ use tracing::Span;
 
 /// A data structure that tracks in-flight requests. It aborts requests,
 /// either on demand or when a request deadline expires.
-#[derive(Debug, Default)]
-pub struct InFlightRequests {
-    request_data: FnvHashMap<u64, RequestData>,
+#[derive(Debug)]
+pub struct InFlightRequests<C> {
+    request_data: FnvHashMap<u64, RequestData<C>>,
     deadlines: DelayQueue<u64>,
 }
 
+impl<C> Default for InFlightRequests<C> {
```

Comment:
This is necessary, because the default Derive macro expects C: Default, but that is not strictly necessary.